Product/Production/Common/Properties/src/main/resources/CORE_X12DSGenericBatchProxyConfig.xml

```diff
@@ -22,8 +22,7 @@ http://www.springframework.org/schema/aop     http://www.springframework.org/sch
     <!--
         Beans defined for adaptercore_x12dsgenericbatchrequest
     -->
-    <alias alias="adaptercore_x12dsgenericbatchrequest"
-           name="adaptercore_x12dsenericbatchrequestwsunsecured" />
+    <alias alias="adaptercore_x12dsgenericbatchrequest" name="adaptercore_x12dsenericbatchrequestwsunsecured" />
```

Comment:
Removed retun(space) between alias and name in Core_x12RealTimeProxyConfig.xml and Core_X12DSGenericBatchProxyConfig.xml which was causing to update realtime adapter from secured to unsecred.